tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -1865,12 +1878,16 @@ Status Remapper::Optimize(Cluster* cluster, const GrapplerItem& item,
       continue;
     }
 
-// TODO(intel-tf):
-// Remove this once TF-MKL supports _FusedConv2D with these operations.
-#ifndef INTEL_MKL
+    // TODO(intel-tf):
+    // Remove this once TF-MKL supports _FusedConv2D with these operations.
     // Remap Conv2D+FusedBatchNorm into the _FusedConv2D;
+#ifdef INTEL_MKL
+    bool is_onednn_enabled = IsMKLEnabled();
+#else
+    bool is_onednn_enabled = false;
+#endif
     ContractionWithBatchNorm contract_with_batch_norm;
-    if (allow_non_differentiable_rewrites &&
+    if (!is_onednn_enabled && allow_non_differentiable_rewrites &&
```

Comment:
Shouldn't this be `is_onednn_enabled`? I.e., we should only try this rewrite when oneDNN is enabled.
```suggestion
    if (is_onednn_enabled && allow_non_differentiable_rewrites &&
```